kasa/experimental/sslaestransport.py

```diff
@@ -406,19 +406,41 @@ async def perform_handshake1(self) -> tuple[str, str, str]:
         _LOGGER.debug(msg)
         raise AuthenticationError(msg)
 
-    def _handshake_session_expired(self):
-        """Return true if session has expired."""
-        return (
-            self._session_expire_at is None
-            or self._session_expire_at - time.time() <= 0
+    async def try_send_handshake1(self, username: str, local_nonce: str) -> dict:
+        """Perform the handshake."""
+        _LOGGER.debug("Will to send handshake1...")
+
+        # Device needs the content length or it will response with 500
```

Comment:
I don't see content length being added here, is this just a general comment? If yes, perhaps worth documenting it in the module docstring?

Reply:
This was an erroneous comment from the other transport, now deleted.